### DISCUSSION TITLE
Any way to run the programs installed without opening whisky first?

### DISCUSSION THREAD

**Discussion by SpaghettiiQ** [Whisky-App/Whisky#253] [Q&A]
Is there a way for me to run a game i installed from outside of whisky's ui?
I tried to make an alias of the exe but that does not work.
Thanks

**Comment by Crd5** [comment]
If you used same wine prefix - yes

**Reply by SpaghettiiQ** [reply]
so how do i do it?

**Reply by Crd5** [reply]
Wait. How did you installed it without Whisky?

**Reply by SpaghettiiQ** [reply]
what do you mean? i installed games normally with whisky, i want to run them from a shortcut so i can add it to my dock, instead of having to open whisky first, similarly to crossover
